src/windowmanagement/WindowLayout/WindowLayout.tsx

```diff
@@ -55,19 +55,29 @@ export function WindowLayout() {
   const { ref } = useWindowLayoutProvider();
   const [visibleMenuItems, setVisibleMenuItems] = useState<string[]>([]);
 
-  const headless: TabGroup = {
-    floatable: false,
-    maximizable: false,
-    tabLocked: true,
-    widthFlex: 1
-  };
-  const regularWindow: TabGroup = {
-    maximizable: false,
-    heightFlex: 1
-  };
-  const groups = {
-    headless,
-    regularWindow
+  const groups: { [key: string]: TabGroup } = {
+    // This is the rc-dock group configuration we use for the transparent window in the
+    // center
+    headless: {
+      floatable: false,
+      maximizable: false,
+      tabLocked: true,
+      widthFlex: 1
+    },
+    // This is the rc-dock group configuration for all the other windows. Note that
+    // 'card' is a pre-existing rc-dock group that adds certina styling to the tab
+    card: {
```

Comment:
I'm wondering if we should some how have a group for our regular windows as well. It seems a bit hacky to me to just modify the `card` group...

Like `"card regularWindow":`  ?

Reply:
I initially had the same thought, but became a bit unsure since "headless" (i.e. the "non-regular" window) is also a predefined rc-dock group. So then, maybe we should do something similar for that one as well. 

I ended up just using "card" since I thought that that was easier to understand than the double class names. But let's discuss what to do here :)